transmit/transmit.go

```diff
@@ -97,7 +97,12 @@ func (d *DefaultTransmission) EnqueueEvent(ev *types.Event) {
 		WithString("api_host", ev.APIHost).
 		WithString("dataset", ev.Dataset).
 		Logf("transmit sending event")
-	libhEv := d.builder.NewEventSized(len(ev.Data))
+	// Calculate approximate size for the libhoney event
+	dataSize := 0
+	for range ev.Data.All() {
```

Comment:
Can we use the updated `ev.GetDataSize()` that resolves back to the new Payload's `GetDataSize` here?

Reply:
Yes, good catch. GetDataSize() was a late addition.